manifests/ca.pp

```diff
@@ -0,0 +1,119 @@
+# == Class: certs
+# Sets up the CA for Katello
+class certs::ca (
+  $pki_dir                 = $::certs::pki_dir,
+  $default_ca_name         = $::certs::default_ca_name,
+  $server_ca_name          = $::certs::server_ca_name,
+  $ca_common_name          = $::certs::ca_common_name,
+  $country                 = $::certs::country,
+  $state                   = $::certs::state,
+  $city                    = $::certs::city,
+  $org                     = $::certs::org,
+  $org_unit                = $::certs::org_unit,
+  $ca_expiration           = $::certs::ca_expiration,
+  $generate                = $::certs::generate,
+  $deploy                  = $::certs::deploy,
+  $server_cert             = $::certs::server_cert,
+  $ssl_build_dir           = $::certs::ssl_build_dir,
+  $group                   = $::certs::group,
+  $katello_server_ca_cert  = $::certs::katello_server_ca_cert,
+  $katello_default_ca_cert = $::certs::katello_default_ca_cert,
+  $nss_db_dir              = $::certs::nss_db_dir,
+  $ca_key                  = $::certs::ca_key,
+  $ca_cert                 = $::certs::ca_cert,
+  $ca_cert_stripped        = $::certs::ca_cert_stripped,
+  $ca_key_password         = $::certs::ca_key_password,
+  $ca_key_password_file    = $::certs::ca_key_password_file,
+) inherits certs {
```

Comment:
This file could be a "private" standalone class that is not designed to be called from other modules. Like ```certs:install``` and ```certs::config```